docker/images.json

```diff
@@ -1,15 +1,24 @@
 {
```

Comment:
Could we also add `m3aggregator` to this btw?  Noticed we don't have an image (can do this in followup I suppose though).

Reply:
Will do